packages/frontend/components/api/RequestBuilder.tsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react'
+import {
+  Box,
+  Button,
+  HStack,
+  Input,
+  Text,
+  Tooltip,
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+} from '@chakra-ui/react'
+import { BigNumber } from 'ethers'
+import { formatFixed } from '@ethersproject/bignumber'
+import { useContractFunction, TransactionState } from '@usedapp/core'
+import { ContractId } from '../../conf/config'
+import { Error } from '../../components/Error'
+import { useContract } from '../../hooks/useContract'
+// @ts-ignore
+import { APIRequestBuilder } from 'types/typechain'
+
+const defaultMultiplier = '1000000000000000000'
+const defaultURL =
+  'https://min-api.cryptocompare.com/data/pricemultifull?fsyms=ETH&tsyms=USD'
+const defaultPath = 'RAW.ETH.USD.VOLUME24HOUR'
+const urlR =
+  /^(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_+.~#?&//=]*)/
+const pathR = /^([A-Za-z][\w$]*(\.[\w$]+)?(\[\d+])?)$/
+const multiR = /^(1(0)*)$/
+
+export const RequestBuilder = (): JSX.Element => {
+  const [url, setURL] = React.useState(defaultURL)
```

Comment:
```suggestion
  const [url, setURL] = useState(defaultURL)
```

Please import all hooks from `react` instead.